### DISCUSSION TITLE
Best practices or most reliable setup.

### DISCUSSION THREAD

**Discussion by cahiatt** [tgorgdotcom/locast2plex#191] [General]
Just wondering for anyone that has experimented with locast2plex on different hosts or formats, what have you found to be the most reliable?

I originally installed locast2plex in Docker on a Server 2012r2 bare metal server. Plex server with plex pass was on the same bare metal machine. At least once a day I would try to tune into a channel and get a "could not tune to channel" error on Plex. I would restart Docker and all was well again.

I've sinced built a new server running Server 2019 Datacenter with docker on the bare metal to start testing. Plex on the metal also. I get the same error about once a day. Completely random times.

Anyone seen an issue similar to this on certain setups?

**Comment by CTJohnK** [comment]
I've used the following configurations, all without issue -

1. Plex and L2P (0.4.2) loaded on a Raspberry Pi 4.
2. Plex loaded on a Synology DS920+ pointing to L2P (0.4.2) loaded on a Raspberry Pi 4.  L2P on the Pi was recently shut down in favor of...
3. Plex loaded on a Synology DS920+ pointing to L2P (0.6.3) )Docker container also on the Synology DS920+.

As reference, here are the steps I used to install L2P for Docker.... https://github.com/tgorgdotcom/locast2plex/discussions/172

**Reply by cahiatt** [reply]
Well it seems Docker does NOT like bare metal or Hyper-V backups.

I moved Docker to a drive that isn't backed up and excluded the VM it creates from Hyper-V backups and it hasn't stalled out since.

**Comment by jtashiro** [comment]
My setup which has been operating smoothly for some time...
Plex Server 
L2P 0.6.4
Silicon Dust Quatro and Dual Connect OTA tuners (before I found Locast2Plex)
Ubuntu 20 server install
bare metal HP core i7 3.3 ghz, 24 GB RAM, 24 TB disk.
